Cargo.toml

```diff
@@ -14,6 +14,7 @@ keywords = ["game", "engine", "gamedev", "graphics", "bevy"]
 license = "MIT OR Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/bevyengine/bevy"
+resolver = "2"
```

Comment:
This will do nothing for crates depending on Bevy and it doesn't seem needed here?

Reply:
I thought it was required for WGPU to build, but I'll have to double-check.

Reply:
[The resolver v2](https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2) let's you specify features per target. Maybe it's needed internally by wgpu? That would be unfortunate...

Reply:
Yes WGPU needs it to compile now. I do remember it was a little difficult to make things work before without the new feature resolver, but it did work. If it is a problem for Bevy, we could discuss it with the WGPU folks, maybe. I don't know if it would be possible to go back to not needing it.

Reply:
`resolver = "2"` will become the default in the 2021 edition.

Reply:
As a dependency can't specify to use the new resolver for the dependent crate, it's a little painful for now for libs to require it... This will change with edition 2021 🎉